modules/nodes-containers-dev-fuse-configuring.adoc

```diff
@@ -11,56 +11,66 @@ As an alternative to the virtual filesystem, you can configure the `/dev/fuse` d
 .Procedure
 
 . Create the pod.
-+
+
 [source,terminal]
 ----
-$ oc exec -ti no-priv -- /bin/bash
+oc apply -f podman-pod.yaml
```

Comment:
Include the `$` prompt [per the doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference). 
```suggestion
$ oc apply -f podman-pod.yaml
```

Reply:
Where does the podman-pod.yaml come from?